migrations/2_deploy_DAOstack.js

```diff
@@ -55,7 +55,7 @@ module.exports = async function(deployer) {
     // apparently we must wrap the first deploy call in a then to avoid
     // what seem to be race conditions during deployment
     // await deployer.deploy(GenesisScheme)
-    deployer.deploy(GenesisScheme, {gas: 6500000}).then(async function(){
+    deployer.deploy(GenesisScheme, {gas: 6900000}).then(async function(){
```

Comment:
As of today, migrations/2_deploy_DAOstack.js is to be moved to arc-js.  See https://github.com/daostack/daostack/pull/223.